src/emulate.c

```diff
@@ -81,6 +88,13 @@ static inline void update_time(riscv_t *rv)
     rv->csr_time[1] = t >> 32;
 }
 
+#if RV32_HAS(SYSTEM)
+static void get_time_now(struct timeval *tv)
+{
+    rv_gettimeofday(tv);
+}
+#endif
```

Comment:
Do we really need to add this function? How about letting the user call `rv_gettimeofday()` directly?

Reply:
> Do we really need to add this function? How about letting the user call `rv_gettimeofday()` directly?

Agree.